src/commands/trialApp/createTrialApp.ts

```diff
@@ -7,5 +7,5 @@ import { IActionContext } from 'vscode-azureextensionui';
 import { openUrl } from '../../utils/openUrl';
 
 export async function createTrialApp(_context: IActionContext): Promise<void> {
-    await openUrl('https://code.visualstudio.com/tryappservice/');
+    await openUrl('https://code.visualstudio.com/tryappservice/?utm_source=appservice-extension');
```

Comment:
We usually use the name in package.json to identify ourselves, aka "vscode-azureappservice"

Reply:
Matt said to use the same value that we use [here](https://github.com/microsoft/vscode-azureappservice/blame/master/README.md#L21) in the README.

Reply:
Makes sense to just change the README unless there's a specific reason we have to use that string.